config/.default/canopy.default.json

```
@@ -68,5 +68,21 @@
   "theme": {
     "defaultTheme": "light",
     "toggleEnabled": false
+  },
+  "cloverOptions": {
+    "canvasBackgroundColor": "var(--colors-slate4)",
+    "canvasHeight": "600px",
+    "openSeadragon": {
+      "gestureSettingsMouse": {
+        "scrollToZoom": false
+      }
+    },
+    "informationPanel": {
+      "open": false,
+      "renderAbout": false,
+      "renderToggle": false
+    },
+    "showTitle": false,
+    "showIIIFBadge": false
```

I'm not in love with having a vendor specific key be at the top of the config. We have multiple vendors (FlexSearch, Leaflet) integrated into our config now, but we root those within the more generic `search` and `map` properties. Though, not convinced `viewer` is the best name for this, I'm curious about your thoughts.

```suggestion
  "viewer": {
    "cloverViewer": {
      "canvasBackgroundColor": "var(--colors-slate4)",
      "canvasHeight": "600px",
      "openSeadragon": {
        "gestureSettingsMouse": {
          "scrollToZoom": false
        }
      },
      "informationPanel": {
        "open": false,
        "renderAbout": false,
        "renderToggle": false
      },
      "showTitle": false,
      "showIIIFBadge": false
    }
```